decidim/decidim
# Fix display condition answer option

#### :tophat: What? Why?
When in a Questionnaire you create a Question with a Display Condition, you select a question that have 2 choices (Single option or Multiple options) and set the condition "Equal to" the dropdown list of the Answer option is empty.

#### :pushpin: Related Issues
- Related to issue https://github.com/decidim/decidim/issues/10374

#### Testing
1. Create a Questionnaire Template
2. Create a first mandatory question of Type Matrix Single option with 2 answer choices (cf screenshot one)
3. Create a 2nd Question with a display condition.
4. For the condition you select the 1st question and select "Equal to"
5. Click on the Answer option dropdown list
6. See that the dropdown list contains the 2 answer options of the first question (cf screenshot two)


### :camera: Screenshots
ONE =>
<img width="1020" alt="one" src="https://github.com/user-attachments/assets/323f20d7-cc5a-4617-ba34-31ea7b0d40e7">

TWO =>
<img width="1004" alt="two" src="https://github.com/user-attachments/assets/45d0aa4a-d1d5-42c7-bd3d-373aa72801c6">

:hearts: Thank you!
